app/views/mailers/notify_mailer/tag_moderator_confirmation_email.text.erb

```diff
@@ -22,7 +22,7 @@ Tag moderation is something we're continuously iterating on, so you can expect a
 We regularly send out a biweekly Mod Newsletter to keep mods up to date on these changes, so look out for it!
 Of course, don't hesitate to reach out to us at any moment if you have any feedback — feel free to write to <%= SiteConfig.email_addresses[:contact] %> and share your thoughts.
 
-Thanks so much for volunteering your time to benefit the <%= community_qualified_name %>. We sincerely appreciate your help!
+Thanks so much for volunteering your time to benefit <%= community_name %>. We sincerely appreciate your help!
```

Comment:
Prior to this PR and the previous PR, this read (in a few places):
```
the <%= community_name %> Community
```
Do we have a preference for which way it should be written? I think that the way it is currently written is consistent with the way it is written in other places in the codebase and reads well, but I'm open to suggestions!